sdk/python/tests/client/test_codegen.py

```diff
@@ -59,7 +59,7 @@ def test_format_name(graphql, expected):
 opts = InputObject(
     "Options",
     fields={
-        "key": InputField(NonNull(Scalar("CacheID"))),
```

Comment:
Are these tests running in CI? Just curious cause I thought this rename happened a while back (but maybe there's some other reason it's needed now)

Reply:
Not actually needed here, just housekeeping.

These are harmless since they're not the same objects that are codegen'ed, they're different subclasses declared at the top of this file that just happen to have the same names as the ones we use. This is just testing if codegen is producing what's expected.